lib/state/appState.js

```diff
@@ -68,6 +68,7 @@ appState.getInitial = function() {
 
   return {
     dropMenu: false,
+    howToUpdateKBLink: 'https://tidepool-project.helpscoutdocs.com/article/6-how-to-install-or-upgrade-the-tidepool-uploader-gen',
```

Comment:
:) Okay, how about pushing this out now and adding a new card to move to a redirecting url later? 

> On Aug 13, 2015, at 10:39 AM, Jana Beck <notifications@github.com> wrote:
> 
> In lib/state/appState.js <https://github.com/tidepool-org/chrome-uploader/pull/139#discussion_r37002361>:
> 
> > @@ -68,6 +68,7 @@ appState.getInitial = function() {
> >  
> >    return {
> >      dropMenu: false,
> > +    howToUpdateKBLink: 'https://tidepool-project.helpscoutdocs.com/article/6-how-to-install-or-upgrade-the-tidepool-uploader-gen',
> If so, we need to set this up. Are you volunteering to spend time wrangling whatever needs to be done (probably mucking around with the WP site) to do that now? We need to get this out as fast as possible, no matter what kind of link is in it (as there is no link at all currently), and we have yet to decide what level of testing process we're comfortable with.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/tidepool-org/chrome-uploader/pull/139/files#r37002361>.
>